sublime-text-merlin.py

```diff
@@ -369,7 +369,7 @@ def on_query_completions(self, view, prefix, locations):
             for r in result['entries']:
                 name = clean_whitespace(r['name'])
                 desc = clean_whitespace(r['desc'])
-                self.cplns.append(((name + '\t' + desc), name))
+                self.cplns.append(((name + '\t' + desc), make_snippet(name, desc)))
```

Comment:
Is there a way we can create two separate autocomplete commands? One which autocomplete with the snips and another without?

Sometimes I just want to complete the name of a function that's already got the args layed out.